implementations/rust/ockam/ockam_command/src/node/create/background.rs

```diff
@@ -38,11 +38,19 @@ impl CreateCommand {
 
         // Run foreground node in a separate process
         // Output is handled in the foreground execution
-        let handle = spawn_node(&opts, cmd)?;
+        let mut handle = spawn_node(&opts, cmd)?;
 
         tokio::select! {
-            _ = wait_while_draining_output(&opts, handle) => { std::process::exit(1) }
-            _ = node_callback.wait_for_signal() => {}
+            res = handle.wait() => {
```

Comment:
I guess that it's difficult to refactor that code with the same code for the authority node?

Reply:
done!